src/file-upload/file-upload.scss

```diff
@@ -36,7 +36,6 @@
       opacity: 0;
       overflow: hidden;
       position: absolute;
-      z-index: -1;
```

Comment:
In the PR description you wrote:
> the element is really small and I'm not sure why it isn't 0 x 0.

If I remember correctly this was an accessibility or SEO based decision.  I tried to find the original source that helped us make our decision but [this is the best I am finding](https://kittygiraudel.com/2021/02/17/hiding-content-responsibly/#the-width-0-and-height-0-declarations) so far.

We may be able to replace lines 34-38 with `@include visually-hidden;` instead.  Pretty sure that mixin did not exist when we created the file upload component.